examples/blaze/src/main/scala/com/example/http4s/blaze/ClientExample.scala

```diff
@@ -11,7 +11,7 @@ object ClientExample {
 
     val client = org.http4s.client.blaze.defaultClient
 
-    val page: Task[String] = client(uri("https://www.google.com/")).as[String]
+    val page: Task[String] = client.getAs[String](uri("https://www.google.com/"))
```

Comment:
This is the only part that makes me sad: I liked that we had a symmetry between the `Request` and `Response` as far as the `.as[T]` syntax. Technically we still do, but its not the 'right' way to do it.
However, it seems a small price to pay for not taking all the socket addresses.